pai-management/src/hadoop-run/image.yaml

```diff
@@ -17,5 +17,11 @@
 
 prerequisite: base-image
 
+copy-list:
+  # TODO It's a bit tricky here.
```

Comment:
>TODO [](start = 4, length = 4)

1. add a comment saying sth. like: please comment the two lines below if you would like to skip hadoop build and use the image published in docker.io
2. please explain exactly what are things todo here. what makes you uncomfortable and wish to change in the future.